atc/atccmd/command.go

```diff
@@ -401,6 +402,16 @@ func (cmd *RunCommand) Runner(positionalArguments []string) (ifrit.Runner, error
 		return nil, err
 	}
 
+	concurrentRequestPolicy := wrappa.NewConcurrentRequestPolicy(
+		cmd.ConcurrentRequestLimits,
+		[]string{atc.ListAllJobs},
+	)
```

Comment:
let's create a `concurrentRequestPolicy()` method on `RunCommand` for this, so we can pull the whole validation block into the `RunCommand.validate()` method.

Reply:
It means we'd construct the policy twice during startup, but it's super cheap, and it would mean we can avoid polluting these private methods with extra parameters.